modules/ROOT/pages/api-products-modifying-products.adoc

```diff
@@ -0,0 +1,70 @@
+= Modifying API Products
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirement changes, you might want to modify a product by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service.
+
+The following tasks are covered in this section:
+
+* <<api-products-creating-products, Adding API instances to Products>>
+* <<removing-api-instances-from-products, Removing API instances from a Product>>
+* <<creating-new-versions-of-products, Creating new versions of a Product>>
+
+[[adding-api-instances-to-products]]
+== Adding API instances to Products
+You might want to add API intances to Products that belong in the same business group of organzations.
+
+=== To add a new API instance to a Product:
+. From the left menu in API Manager, click Products.
+. From the Products list on the right details pane, expand the Product to view the version.
+. Click the version of the Product to which you want to add the API instance.
+. From the top-right corner of the window, click Edit.
+The Adding API Instances window appears.
++
+image::add-api-instances.png[Adding API Instances]
++
+. If the API instance and the Product belong in the same business group or organization, select the environment, API instance, and version that you want to add to the Product.
+. If the API instance and Product belong in different business groups:
+.. Click Add Organization.
+.. Select the organization or business group in which your API instance is available.
+.. From the environment, select the API instance, and version that you want to add.
+. Click Save.
+If your Product is published in Exchange, your Exchange asset reflects the  new API instance. To publish your product in Exchange, see xref:api-products-publishing-to-exchange.adoc[Publishing API Products to Exchange].
+
+[[removing-api-instances-from-products]]
+== Removing API instances from Products
+If your business requirement changes, you might want to modify a product by removing an existing API instance.
+
+=== To remove an API instance from a Product:
+. From the left menu in API Manager, click Products.
+. From the Products list on the right details pane, expand the Product to view the version.
+. From the top-right corner, click Edit.
+. Click the garbage-can icon next to the API instance that you want to remove. 
```

Comment:
The icon was changed for a cross now (same for the business group deletion)

Reply:
Done! Thanks!